README.md

```
@@ -10,6 +10,9 @@ Guide for adding metrics can be found [here](docs/adding-metrics-guide.md)
 
 Some tips to find issues while developing can be found [here](docs/development-tips.md)
 
+Memory monitoring — the `/admin/memory_stats` dashboard and the `MemDbg`
+console toolkit — is described [here](docs/memory-monitoring.md)
```

markdownlint-cli2 (0.23.0)

[warning] Link text should be descriptive

(MD059, descriptive-link-text)